expressEngine/product/service/product.js

```diff
@@ -25,6 +25,15 @@ class Service {
     }
     return { data: product, message: success.productRegistered };
   }
+
+  updateProduct = async (data) => {
+    const { filter, dataToUpdate } = data;
+    const product = await DBService.updateOne(Product, dataToUpdate, filter);
+    if (product.error) {
+      return product;
+    }
+    return { data: product, message: success.productUpdated };
```

Comment:
{
    "data": {
        "n": 1,
        "nModified": 1,
        "ok": 1
    },
    "message": "Product updated successfully"
}
@aman-successive  this is the output, doesn't look good.

Reply:
Success handler was created to accept both data and message,
Should we not include data for success message then we get the message "data": "true" which is meaningless

Reply:
@aman-successive it is not compulsory to pass data, that's why it has a default value.
data: 'true' is not meaningless it shows success and even look prettier than this, not understandable data.

Reply:
If you read the docs, you will understand what each of the data fields here means
Since I have worked on the update functionality of MongoDB, I can easily tell about the status of the operation just by looking at this data.

Reply:
@aman-successive I understand these fields, so I will suggest using these fields in a better way.

Reply:
@aman-successive Please add the link of the docs. So that we can also read it.

Reply:
https://docs.mongodb.com/manual/reference/method/db.collection.updateOne/#db.collection.updateOne

Reply:
@chiragSuccessive Added changes to response data

Reply:
@aman-successive ok